### PR TITLE
Attempt to fix EVT_HS_AuthedByPassword warnings in HA log

### PR DESCRIPTION
**Pull request by jghaanstra**
Attempt to fix EVT_HS_AuthedByPassword warnings in HA log

### CODE REVIEW

**Comment by Kane610** [comment]
Thanks! Would you mind adding the full log message to the description? Helps should additional development be warranted down the line.

**Comment by jghaanstra** [comment]
Sorry for not responding sooner. I had to reproduce the error and it got of my radar. Just did and this is the full error.

```
Unsupported event key EVT_HS_AuthedByPassword
Unsupported event {'guest': '1a:xx:xx:xx:xx:f6', 'key': 'EVT_HS_AuthedByPassword', 'subsystem': 'wlan', 'is_negative': False, 'site_id': '5xxxxxxxxxxxxxxxxxxd', 'time': 1682358988725, 'datetime': '2023-04-24T17:56:28Z', 'msg': 'Guest[1a:xx:xx:xx:xx:f6] is authorized by password', '_id': '6xxxxxxxxxxxe'}
```

File: aiounifi/models/event.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ class EventKey(enum.Enum):
     AD_SCHEDULE_UPGRADE_FAILED_NOT_FOUND = "EVT_AD_ScheduleUpgradeFailedNotFound"
 
     HOT_SPOT_VOUCHER_USED = "EVT_HS_VoucherUsed"
+    HOT_SPOT_AUTHED_BY_PASSWORD = "EVT_HS_AuthedByPassword"
```
Comment (Kane610) [inline aiounifi/models/event.py:99]:
```suggestion
    HOT_SPOT_AUTHED_BY_PASSWORD = "EVT_HS_AuthedByPassword"
    HOT_SPOT_VOUCHER_USED = "EVT_HS_VoucherUsed"
```

**Comment by Kane610** [comment]
PR is up, should be a part of 2023.5 release

https://github.com/home-assistant/core/pull/91966

**Pull request merged**